File: src/main/resources/output/IFC4X3_csharp/Entities/IfcResourceLevelRelationship/IfcPropertyDependencyRelationship/IfcPropertyDependencyRelationship.cs
public class IfcPropertyDependencyRelationship : IfcResourceLevelRelationship
{
    public IfcProperty DependingProperty { get; set; }
    public IfcProperty DependantProperty { get; set; }
    public IfcText Expression { get; set; }

    // === WHERE CLAUSES ===
    // NoSelfReference : DependingProperty :<>: DependantProperty
}

File: src/main/resources/output/IFC4X3_js/Entities/IfcResourceLevelRelationship/IfcPropertyDependencyRelationship/IfcPropertyDependencyRelationship.js
class IfcPropertyDependencyRelationship extends IfcResourceLevelRelationship {
    constructor() {
        /** @type {IFCPROPERTY} */
        this.DependingProperty = null;
        /** @type {IFCPROPERTY} */
        this.DependantProperty = null;
        /** @type {IFCTEXT} */
        this.Expression = null;
    }

    // === WHERE CLAUSES ===
    // NoSelfReference : DependingProperty :<>: DependantProperty
}
